tests/ActivityloggerTest.php

```diff
@@ -243,4 +244,59 @@ public function it_returns_an_instance_of_the_activity_log_after_logging_when_us
 
         $this->assertInstanceOf($activityClassName, $activityModel);
     }
+
+    /** @test */
+    public function it_uses_any_guards_user_used()
+    {
+        Auth::logout();
+
+        $employeeID = 1;
+        $a = [
```

Comment:
I do not like variables that have no meaning in their name. Could you rename this. 

I'd like the `config` to be set with as little calls possible. How it is set up now is quite fragmented and hard to follow.